egs/fisher_callhome_spanish/asr1/run.sh

```diff
@@ -0,0 +1,347 @@
+#!/bin/bash
+
+# Copyright 2017 Johns Hopkins University (Shinji Watanabe)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+. ./path.sh
+. ./cmd.sh
+
+# general configuration
+backend=chainer
+stage=0        # start from 0 if you need to start from data preparation
+ngpu=0         # number of gpus ("0" uses cpu, otherwise use gpu)
+debugmode=1
+dumpdir=dump   # directory to dump full features
+N=0            # number of minibatches to be used (mainly for debugging). "0" uses all minibatches.
+verbose=0      # verbose option
+resume=        # Resume the training from snapshot
+seed=1
+
+# feature configuration
+do_delta=false
+
+# network architecture
+# encoder related
+etype=vggblstmp     # encoder architecture type
+elayers=6
+eunits=320
+eprojs=320
+subsample=1_2_2_1_1 # skip every n frame from input to nth layers
+# decoder related
+dlayers=1
+dunits=300
+# attention related
+atype=location
+adim=320
+awin=5
+aheads=4
+aconv_chans=10
+aconv_filts=100
+
+# hybrid CTC/attention
+mtlalpha=0.2
+
+# label smoothing
+lsm_type=unigram
+lsm_weight=0.05
+
+# minibatch related
+batchsize=30
+maxlen_in=800  # if input length  > maxlen_in, batchsize is automatically reduced
+maxlen_out=150 # if output length > maxlen_out, batchsize is automatically reduced
+
+# optimization related
+opt=adadelta
+epochs=15
+
+# rnnlm related
+use_wordlm=true     # false means to train/use a character LM
+lm_vocabsize=65000  # effective only for word LMs
+lm_layers=1         # 2 for character LMs
+lm_units=1000       # 650 for character LMs
+lm_opt=sgd          # adam for character LMs
+lm_batchsize=300    # 1024 for character LMs
+lm_epochs=20        # number of epochs
+lm_maxlen=40        # 150 for character LMs
+lm_resume=          # specify a snapshot file to resume LM training
+lmtag=              # tag for managing LMs
+
+# decoding parameter
+lm_weight=1.0
+beam_size=30
+penalty=0.0
+maxlenratio=0.0
+minlenratio=0.0
+ctc_weight=0.3
+recog_model=model.acc.best # set a model to be used for decoding: 'model.acc.best' or 'model.loss.best'
+
+# scheduled sampling option
+samp_prob=0.0
+
+# data
+wsj0=/export/corpora5/LDC/LDC93S6B
+wsj1=/export/corpora5/LDC/LDC94S13B
+
+# exp tag
+tag="" # tag for managing experiments.
+
+. utils/parse_options.sh || exit 1;
+
+. ./path.sh
+. ./cmd.sh
+
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+train_set=train_si284
+train_dev=test_dev93
+train_test=test_eval92
+recog_set="test_dev93 test_eval92"
+
+if [ ${stage} -le 0 ]; then
+    ### Task dependent. You have to make data the following preparation part by yourself.
+    ### But you can utilize Kaldi recipes in most cases
+    echo "stage 0: Data preparation"
+    local/wsj_data_prep.sh ${wsj0}/??-{?,??}.? ${wsj1}/??-{?,??}.?
```

Comment:
I only find data prep for WSJ, you need to add instructions for this recipe.

Reply:
Thanks @Fhrozen. I will update the above suggestions. Shinji's suggestions and some of the above are done from the recent commit ef46731.